.github/workflows/aws-deploy.yaml

```diff
@@ -247,6 +247,40 @@ jobs:
         run: |
           cd sceptre/scicomp
           sceptre launch prod --yes
+  sceptre-strides:
+    if: github.event_name == 'push'
+    needs: org-formation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Cache python modules
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install sceptre sceptre-ssm-resolver
+          pip install git+git://github.com/Sceptre/sceptre-file-resolver.git
+      - name: Assume AWS Role
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
```

Comment:
What IAM entity in stridespool is this key associated with?

Reply:
this is our service account in our organizations account to access roles in other accounts. This is the cross-account access technique.